deepdrivemd/sim/openmm_stream/run_openmm.py

```diff
@@ -0,0 +1,265 @@
+import shutil
+from pathlib import Path
+from typing import Optional
+import simtk.unit as u
+import simtk.openmm as omm
+import simtk.openmm.app as app
+from mdtools.openmm.sim import configure_simulation
+from deepdrivemd.utils import Timer, parse_args
+from deepdrivemd.data.api import DeepDriveMD_API
+from deepdrivemd.sim.openmm.config import OpenMMConfig
+from openmm_reporter import ContactMapReporter
+import sys
+import os
+# from deepdrivemd.misc.OutlierDB import OutlierDB
+from OutlierDB import *
+from lockfile import LockFile
+import pickle
+import parmed as pmd
+import numpy as np
+import subprocess
+import glob
+
+class SimulationContext:
```

Comment:
Can import this SimulationContext from non-streaming version, if they are the same.

Reply:
I did not touch it. At first glance it is not obvious what I need from it or what is its purpose. It looks like it is a wrapper around the configuration that does some extra checking/calculation of parameters.

Apparently there are some things there that I am not using...  I guess, I need to figure out what functions are really called as the program runs. I suspect I can clean a lot from it. 

It would be nice to have a documentation that explains your design. I am still not sure I understand why the code is organized the way it is organized.

Reply:
For now reused SimulationContext from non-streaming version since at the moment I do not have any special use for it to customize.

Reply:
Sorry, another comment that got buried. This was an effort (albeit not very general one) to abstract some of the I/O and file handling concerns from the main logic of the simulation. I think the only place it is used in your code is to handle the PDB file inputs. It also provides some useful support for getting the associated topology files. That will be helpful when we generalize the streaming version to the protein-ligand complex use case. For now though, it is not really needed in the current code. It's fine to use it as you are if it works, no need to remove it.

Documentation is on the way. We are designing a more general API for MD, ML, Agent, etc tasks that should really help to make sense of things.